ddlambda.go

```diff
@@ -42,54 +43,65 @@ type (
 		// Site is the host to send metrics to. If empty, this value is read from the 'DD_SITE' environment variable, or if that is empty
 		// will default to 'datadoghq.com'.
 		Site string
-
 		// DebugLogging will turn on extended debug logging.
 		DebugLogging bool
 		// EnhancedMetrics enables the reporting of enhanced metrics under `aws.lambda.enhanced*` and adds enhanced metric tags
 		EnhancedMetrics bool
+		// DDTraceEnabled enables the Datadog tracer.
+		DDTraceEnabled bool
```

Comment:
nit: Probably doesn't need to be prefixed with DD since it's on the config object.

Reply:
Actually I think it's better to leave it like this. This enables **Datadog** tracing -- as opposed to X-Ray.

Reply:
That makes sense